### DISCUSSION TITLE
Symbolic Link in Windows

### DISCUSSION THREAD

**Discussion by aibradford** [pymedusa/Medusa#7347] [Q&A]
I just setup Medusa on Windows.

It cant see the mapped drives. So i followed the instructions to change the account being used... 
"C:\Windows\system32>sc config Medusa obj= "Adminusername" password= "mypassword"
[SC] ChangeServiceConfig FAILED 1057:

The account name is invalid or does not exist, or the password is invalid for the account name specified.
I know for certain they are not incorrect.

So im looking for other ways.

So than i made a symbolic link "mklink \networked\location C:\Tvdrive", and it made the link fine. However, when i enter it into Medusa, it STILL cannot read the files in the linked folder. The windows explorer shows the files fine, but not Medusa. I'm assuming its a permissions issue, but id love some advise.

Can someone assist. All I want to do is use Medusa instead of Sonarr with my files on my NAS. I want to use Medusa because it has a lot more features but I cant get it to see any of my files.

**Comment by Mika3578** [comment]
Same issue for me on windows 10 unable to see mapped folder... change medusa service don't work...
Any recent tutorial to have the capability to medusa support network mapped folders?
